### PR TITLE
[CPP_RPC] export listdir for RPC

### PR DESCRIPTION
**Pull request by Ever-Kid**
This PR register `tvm.rpc.server.listdir` using `ListDir` and export it in python frontend for PRC use.

### CODE REVIEW

File: apps/cpp_rpc/rpc_env.cc [hunk 1]
```diff
@@ -72,6 +72,13 @@ namespace runtime {
  */
 void CleanDir(const std::string& dirname);
 
+/*!
+ * \brief ListDir get the list of files in a directory
+ * \param dirname The root directory name
+ * \return vector Files in directory.
+ */
+std::vector<std::string> ListDir(const std::string& dirname);
```
Comment (wrongtest-intellif) [inline apps/cpp_rpc/rpc_env.cc:80]: Is the implementation missing?

File: apps/cpp_rpc/rpc_env.cc [hunk 2]
```diff
@@ -123,6 +130,15 @@ RPCEnv::RPCEnv(const std::string& wd) {
     *rv = this->GetPath(args[0]);
   });
 
+  TVM_REGISTER_GLOBAL("tvm.rpc.server.listdir").set_body([this](TVMArgs args, TVMRetValue* rv) {
+    std::string dir = this->GetPath(args[0]);
+    std::string ret;
+    for (auto d : ListDir(dir)) {
+      ret.append(d.append(","));
```
Comment (wrongtest-intellif) [inline apps/cpp_rpc/rpc_env.cc:137]: Could we use ostringstream here?

File: python/tvm/rpc/client.py [hunk 3]
```diff
@@ -145,6 +145,18 @@ def remove(self, path):
             self._remote_funcs["remove"] = self.get_function("tvm.rpc.server.remove")
         self._remote_funcs["remove"](path)
 
+    def listdir(self, path):
+        """ls files from remote temp folder.
+
+        Parameters
+        ----------
+        path: str
+            The relative location to remote temp folder.
```
Comment (wrongtest-intellif) [inline python/tvm/rpc/client.py:154]: It would be great to describe the return type and split token.

**Review by wrongtest-intellif — CHANGES_REQUESTED** [review]

File: apps/cpp_rpc/rpc_env.cc (hunk 1 above)
Comment (Ever-Kid) [reply apps/cpp_rpc/rpc_env.cc:80]: already coded downside the same file

**Review by wrongtest-intellif — APPROVED** [review]

**Pull request merged**